oggm/utils/_downloads.py

```diff
@@ -1528,29 +1528,41 @@ def alaska_dem_zone(lon_ex, lat_ex):
     return gdf.tile.values if len(gdf) > 0 else []
 
 
-def copdem_zone(lon_ex, lat_ex):
+def copdem_zone(lon_ex, lat_ex, version='90'):
     """Returns a list of Copernicus DEM tarfile and tilename tuples
     """
 
+    # because we use both meters and arc secs in our filenames...
+    if version == '90':
+        asec = '30'
+    elif version == '30':
+        asec = '10'
+    else:
+        raise InvalidDEMError('COPDEM Version not valid.')
+
     # path to the lookup shapefiles
-    gdf = gpd.read_file(get_demo_file('RGI60_COPDEM_lookup.shp'))
+    df = pd.read_csv(get_demo_file('copdem{}_2021_1.csv'.format(version)))
```

Comment:
Note for a later PR: this may need to be read into a permanent container to not have to read the same file over again each time a glacier wants to know the tiles

Reply:
fair point. if you have any similar case which is working, I could do it

Reply:
`cfg.DATA` is a multiprocessing aware dictionary used for example here:

https://github.com/OGGM/oggm/blob/919d9044fea7c54320d533e67607b2f5d1b7c51a/oggm/utils/_downloads.py#L1316-L1358